proxy/http/HttpConfig.h

```diff
@@ -851,8 +852,8 @@ extern volatile int32_t icp_dynamic_enabled;
 /////////////////////////////////////////////////////////////
 inline HttpConfigParams::HttpConfigParams()
   : proxy_hostname(NULL), proxy_hostname_len(0), server_max_connections(0), origin_min_keep_alive_connections(0),
-    max_websocket_connections(-1), disable_ssl_parenting(0), enable_url_expandomatic(0), no_dns_forward_to_parent(0),
-    uncacheable_requests_bypass_parent(1), no_origin_server_dns(0), use_client_target_addr(0), use_client_source_port(0),
+    max_websocket_connections(-1), disable_ssl_parenting(0), enable_url_expandomatic(0),
+    no_dns_forward_to_parent(0), no_origin_server_dns(0), use_client_target_addr(0), use_client_source_port(0),
```

Comment:
oh, I see. Yeah, I don't think that works, the trailing comma isn't allowed in initializers :-/. So, this is an all or nothing thing, i.e. we'd have to change the .clang-format config file :-/. The other option is to not clang-format these initializers, but I really don't like to use that, because it allows for future mess (because we know we can't format manually).

If we make the change to .clang-format, the diff is large, but not obscene: http://paste.fedoraproject.org/362988/60098146/


I think if we do this though, we should make a separate Jira / PR, right? I'm +1 on this btw.